dev-docs/feature-conditional.md

```diff
@@ -0,0 +1,239 @@
+    1. The second bid requires that "phone" be present in the label array, otherwise it won't be part of the auction.
+
+## Use case: What if some bidders have different parameters for different devices?
+
+For reporting and targeting purposes, Publishers and SSPs sometimes break out different 'placements' for different platforms.
+
+For instance, say that a given bidder wants to define different placements for different devices according to this table:
+
+{: .table .table-bordered .table-striped }
+| Device | Placement ID |
+|--------|---------|
+| Display | 1111 |
+| Phones and tablets | 2222 |
+
+Assuming the same sizeConfig as in the first use case above, the AdUnit would contain bids for both
+placements, but the conditional `labelAny` is added to them both:
+
+{% highlight js %}
+
+var AdUnits = [{
+    code: "ad-slot-1",
+    sizes: [[768,90], [468,60], [320,50]], 
+    bids: [
+        {
+            bidder: "bidderA",
+            labelAny: ["display"],
+            params: {
+                placement: "1111"
+            }
+       },{
+            bidder: "bidderA",
+            labelAny: ["phone", "tablet"],
+            params: {
+                placement: "2222"
+            }
+       }
+   ]
+}]
+
+{% endhighlight %}
+
+Example of how this works:
+
+1. Users with a screen width of 1000px will cause the second mediaQuery to fire.
+1. This rule puts "tablet" into the label array and a list of sizes into sizesSupported.
+1. Then the AdUnit is processed:
+    1. The first bid requires that the label "display" be present in the array. It's not, so that bid is skipped.
+    1. The second bid requires that either "phone" or "tablet" be present. Since tablet is in the label array, that bid is activated and the correct placement is sent to bidderA.
+
+## Use Case: What if some AdUnit auctions should be skipped entirely for some devices?
+
+Say there's a responsive page where one of the AdUnits only supports larger sizes, so it doesn't make sense
+on phones. We really don't want to use that AdUnit for mobile users. Wouldn't it be great if `labelAny` and `labelAll` could be applied to the entire AdUnit? They can. Here's an example:
+
+{% highlight js %}
+
+var AdUnits = [{
+    code: "ad-slot-1",
+    sizes: [[768,90]], 
+    labelAny: ["display", "tablet"], // skip the ad unit entirely for phones
+    bids: [
+        {
+            bidder: "bidderA",  // no conditions
+            params: {
+                placement: "1111"
+            }
+       },{
+            bidder: "tabletBidder",
+            labelAny: ["tablet"], // additional conditions can be applied to bids
+            params: {
+                placement: "2222"
+            }
+       }
+   ]
+}]
+
+{% endhighlight %}
+
+## Use Case: What if some bid requests apply only to users originating certain countries? 
+
+Labels aren't constrained to describing device size -- they can be used for any condition the page wants
```

Comment:
I'm a little curious about the claim "they can be used for any condition the page wants to define".  It seems overly broad.

Would it be closer to correct to say something like "almost any condition ..."?